### ISSUE TITLE
Presets Panel: Missing accessible name for the presets modal

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#5444]
## Issue Description
The  presets panel modal window has no accessible name. It is only described as "dialog."

## WCAG Criteria
[4.1.2 Name, Role, Value - A](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

## Notes
Use `aria-label` to give the dialog a unique, descriptive accessible name. For example, `aria-label="Presets"`.

This issue is referred to in #5187.

## Screenshot or Screen Recording

<img width="1269" alt="Screen shot showing the dialog has no accessible name liste din the computed properties view of developer tools." src="https://github.com/user-attachments/assets/3868f17f-3021-4ba7-acdd-9bc43e976b0c" />

## Relevant Code
```html
<div data-side="bottom" data-align="center" data-state="open" role="dialog" id="radix-:rn:" class="mt-2 max-h-[495px] overflow-x-hidden rounded-lg border border-gray-200 bg-white shadow-lg dark:border-gray-700 dark:bg-gray-700 dark:text-white md:min-w-[400px]" tabindex="-1" style="--rad">
```
